README.md

```diff
@@ -1,8 +1,71 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Mitchell Acton . Jarvis Consulting
 
+Graduated with an Master of Engineering degree in Mechanical and Materials Engineering from the University of Western Ontario, and a Bachelor of Applied Science degree in Nanotechnology Engineering from the University of Waterloo. I gained exposure and experience in data analytics and software development during school, during internships, through online courses, and through work experience after graduating. Eventually this became my main focus at work, so I decided to formally make the switch to a full time software development role with Jarvis. Since starting with Jarvis, I have become comfortable working in an Agile/Scrum development cycle, have gained valuable experience with Linux, PostgreSQL database management, and Java. I'm looking forward to getting the opportunity to put these tools to use along with my existing data analysis and machine learning skills to make high quality software products with significant business impacts.
+
+## Skills
+
+**Proficient:** Java, Python, JavaScript, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Linux/Bash, HTML, Docker, Machine Learning, Computer Hardware/Assembly
+
+**Familiar:** C++, Neural Networks, Networking, Cloud Computing (MS Azure/IBM Watson), Computer Vision
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MitchellActon](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon)
+
+
+**Linux Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MitchellActon/tree/master/linux_sql)]: Developed a Linux cluster hardware resource management tool intended to be used by Linux system administrators. Includes bash and SQL scripts to automatically set up the docker PostgreSQL container and database, initialize tables, and scripts to populate the tables with host hardware specification and resource usage data.
```

Comment:
too short. explain how these technologies work together

e.g. Wrote bash script to collect hardware data .. Persist data into a PostgreSQL database provisioned with Docker ....

![image](https://user-images.githubusercontent.com/58951265/134978863-92fcaa5f-406b-444c-969c-d7fea7aea3b9.png)